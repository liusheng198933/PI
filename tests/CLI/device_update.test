#!/bin/sh
./run_one_test.py $srcdir/testdata device_update swap_1.json
