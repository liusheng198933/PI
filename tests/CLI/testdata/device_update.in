add_p4 swap_2.json bmv2
table_dump t_ingress_1
table_dump t_ingress_2
update_device_start 1 swap_2.json
table_dump t_ingress_1
table_dump t_ingress_2
update_device_end
table_dump t_ingress_1
table_dump t_ingress_2
