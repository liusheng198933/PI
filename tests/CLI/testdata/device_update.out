????
Device assigned successfully.
Selecting device.
????
P4 config added with id 1
????
Successfully retrieved 0 entrie(s).
==========
TABLE ENTRIES
==========
Dumping default entry
EMPTY
==========
????
????
Device update started.
????
????
Successfully retrieved 0 entrie(s).
==========
TABLE ENTRIES
==========
Dumping default entry
EMPTY
==========
????
Device update done.
????
????
Successfully retrieved 0 entrie(s).
==========
TABLE ENTRIES
==========
Dumping default entry
EMPTY
==========
????
